server/@types/migrationImport/index.d.ts

```diff
@@ -148,6 +152,10 @@ export interface paths {
     /** Requires role <b>MIGRATE_ADJUDICATIONS</b> */
     get: operations['getActiveMigrationDetails_5']
   }
+  '/migrate/activities/ids': {
+    /** Searches for active course activities with allocated prisoners (excluding DPS program services). Requires role MIGRATE_ACTIVITIES */
+    get: operations['findActivitiesToMigrate']
+  }
```

Comment:
When getting ids of activities (or allocations) to migrate call these new endpoints on the migration service which understand the business rules when selecting entities to migrate (i.e. don't migrate any of the new DPS program services).